src/utils/context.ts

```diff
@@ -166,67 +165,6 @@ export function sendSlack(message: string): boolean {
   return true;
 }
 
-/**
- * Convenient utility to log in case theres an error writing in the registry and return a boolean with the result of the operation
- *
- * @param registry Tenderly registry
- * @returns a promise that returns true if the registry write was successful
- */
-export async function writeRegistry(): Promise<boolean> {
-  if (executionContext) {
-    return handlePromiseErrors(
-      "Error writing registry. Not more attempts!",
-      backOff(
-        async () => {
-          if (!executionContext) {
-            return undefined;
-          }
-          return executionContext.registry.write();
-        },
-        {
-          numOfAttempts: 10,
-          timeMultiple: 2,
-          retry: (e, attemptNumber) => {
-            console.warn(
-              `Error writing registry. Attempt ${attemptNumber}. Retrying...`,
-              e
-            );
-            return true;
-          },
-        }
-      ).catch((e) => {
-        if (executionContext) {
-          console.error(
-            "Error writing registry. Not more attempts! Dumping the orders",
```

Comment:
why removing the exponential-backoff logic? 

I believe there could still be some write error, right? don't we want to re-attempt?

Reply:
Write retry logic has been removed as there is an assumption now around it having atomic guarantees. If the storage fails, IMO the service should die, which will cause the container to cycle and this will become visible.

As the run loop now does a "warm up", there is no danger in it losing state as sync picks back up where it left off.

Reply:
fine, not a big deal, but even if the operations are atomic, the re-attempt will be also atomic, so no harm on re-attempting and trying to heal the system without another re-run

anyways, i don't think an IO operation on a docker volume will likely fail, so not a big deal. Its just, it was a small improvement and i didn't feel needed to be removed